### PR TITLE
chore: remove redundant line

### PR DESCRIPTION
**Pull request by splincode**

### CODE REVIEW

**Review by vladimirpotekhin — APPROVED** [review]

File: projects/demo/src/modules/app/stackblitz/project-files/src/main.ts.md [hunk 1]
```diff
@@ -24,12 +24,6 @@ bootstrapApplication(Root, {
       provide: 'Pythons',
       useValue: ['John Cleese', 'Eric Idle', 'Michael Palin', 'Graham Chapman', 'Terry Gilliam', 'Terry Jones'],
     },
-    /**
-     * A workaround for StackBlitz only (it does not support assets).
-     * Don't use this approach in real-world applications!
-     */
-    // TODO: remove `next/` after 4.0 release
-    tuiAssetsPathProvider('https://taiga-ui.dev/next/assets/taiga-ui/icons'),
```
Comment (nsbarsukov) [inline projects/demo/src/modules/app/stackblitz/project-files/src/main.ts.md:32]: https://taiga-ui.dev/next/components/avatar#content-types

before-stackblitz

Comment (nsbarsukov) [reply projects/demo/src/modules/app/stackblitz/project-files/src/main.ts.md:32]: Your branch stackblitz

after-stackblitz

**Review by nsbarsukov — CHANGES_REQUESTED** [review]

File: projects/demo/src/modules/app/stackblitz/project-files/src/main.ts.md (hunk 1 above)
Comment (splincode) [reply projects/demo/src/modules/app/stackblitz/project-files/src/main.ts.md:32]: fixed

**Review by mdlufy — APPROVED** [review]

**Comment by splincode** [comment]
@nsbarsukov

**Review by waterplea — APPROVED** [review]

**Review by nsbarsukov — APPROVED** [review]

**Pull request merged**